eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -83,7 +84,8 @@ case class NodeParams(keyManager: KeyManager,
                       routerConf: RouterConf,
                       socksProxy_opt: Option[Socks5ProxyParams],
                       maxPaymentAttempts: Int,
-                      enableTrampolinePayment: Boolean) {
+                      enableTrampolinePayment: Boolean,
+                      instanceId: UUID) {
```

Comment:
I'd put this above in the arg list, maybe right after the `keyManager`. It also deserves a description.